packages/web-data/README.md

```diff
@@ -0,0 +1 @@
+# `@vertex-protocol/web-data`
```

Comment:
a really nice to have but would be nice to show how a frontend can use this package and maybe show an example, etc

Reply:
Yeah - this is one thing we should look into a lot more. I added a [task](https://www.notion.so/vertexprotocol/Public-Documentation-for-using-vertex-protocol-react-client-aa226a97ad2547aaa60065149cd96941?pvs=4). For now, this is extracted so we can split out Blitz